src/st/e_life_up.h

```diff
@@ -233,7 +237,7 @@ void EntityLifeUpSpawn(Entity* self) {
                     self->animFrameIdx = 0;
                 } else {
                     self->entityId = E_EQUIP_ITEM_DROP;
-                    self->pfnUpdate = EntityRelicOrb;
+                    self->pfnUpdate = EntityEquipItemDrop;
```

Comment:
I am not sure how this wasn't failing on PSX? But it's still matching after this fix (which was required on PSP)